src/editor/index.jsx

```diff
@@ -43,6 +44,7 @@ window.addEventListener("message", listenForEvalFramePortReady, false);
 handleServerVariables(store);
 handleInitialIomd(store);
 store.dispatch(restoreLocalAutosave());
+store.dispatch(getFiles());
```

Comment:
Is this something we currently need to do on startup?

I feel like it's worth the extra bit of effort to make this happen when the files modal is opened (or a file source is downloaded), and eliminate the line here. I know this is extra work, but this PR doesn't actually fix the bug in its current form.

Reply:
The purpose of this PR was not to fix the fact that the files modal doesn't make the request, but to prepare for refactoring that tab to do what the file sources tab does, which will require more cycles than I have. I'm happy to add the requesting but there is more UX work than you'd think to make the experience smoother.

Reply:
On second thought, sure, I'll add these in those spots for now. I think the UX weirdness is probably worth having those update.

Reply:
Also to clarify – we still need to dispatch `getFiles` for the programmatic API to work properly, so I'm going to keep this here.

Reply:
You make a good point – this PR should actually do the work necessary to close #2200, which isn't much additional code.